apps/gitness/src/pages/pull-request-list-page.tsx

```diff
@@ -174,16 +172,21 @@ function PullRequestListPage() {
           author: item?.author?.display_name,
           name: item?.title,
           // TODO: fix review required when its actually there
-          reviewRequired: true,
+          reviewRequired: !item?.is_draft,
           merged: item.merged,
           comments: item.stats?.conversations,
           number: item?.number,
+          is_draft: item?.is_draft,
           // TODO: add label information to display associated labels for each pull request
           // labels: item?.labels?.map((key: string, color: string) => ({ text: key, color: color })),
           // TODO: fix 2 hours ago in timestamp
           timestamp: item?.created ? timeAgoFromEpochTime(item?.created) : '',
           source_branch: item?.source_branch,
-          state: item?.state
+          state: item?.state,
+          labels: item?.labels?.map((label, index) => ({
+            text: label?.key && label?.value ? `${label?.key}:${label?.value}` : (label.key ?? ''),
+            color: colorArr[index % colorArr.length]
```

Comment:
ideally color should be fixed per the label. not randomised. 
eg. "do not merge" label is usually always red.